en/operations-selfhosted/docker-containers.html

```diff
@@ -311,3 +311,23 @@ <h2 id="resource-usage">Resource usage</h2>
   and they all listen to work to do, changes in config and so forth.
 </p>
 <p>Therefore, an "idle" container instance consumes CPU and memory.</p>
+
+
+
+<h2 id="troubleshooting">Troubleshooting</h2>
+<p>
+  The Vespa documentation examples use <code>docker</code>.
+  The Vespa Team has good experience with using <code>podman</code>, too,
+  in the examples just change from <code>docker</code> to <code>podman</code>.
+  <a href="https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman">emulating-docker-cli-with-podman</a>
+  is a useful resource.
```

Comment:
Maybe add a note that Podman v5 is recommended for MacOS (uses Apple native hypervisor, which is said to perform much better), or if Nvidia GPU support is desired. https://github.com/containers/podman/blob/main/RELEASE_NOTES.md

Reply:
Good point! added